styx-scheduler-service/src/main/java/com/spotify/styx/api/SchedulerResource.java

```diff
@@ -145,10 +145,10 @@ public SchedulerResource(
     if (!configuration.dockerImage().isPresent()) {
       return Response.forStatus(Status.BAD_REQUEST.withReasonPhrase("Missing docker image"));
     }
-    final Collection<String> errors = dockerImageValidator.validateImageReference(
-        configuration.dockerImage().get());
+    final Collection<String> errors = workflowValidator.validateWorkflowConfiguration(configuration);
     if (!errors.isEmpty()) {
-      return Response.forStatus(Status.BAD_REQUEST.withReasonPhrase("Invalid docker image: " + errors));
+      return Response.forStatus(Status.BAD_REQUEST.withReasonPhrase("Invalid workflow configuration: "
```

Comment:
`Status.BAD_REQUEST` has been statically imported already. Change to `BAD_REQUEST` would fix a few codacy complaints. There are a few in this file.